src/saving/SaveHelper.cs

```diff
@@ -242,6 +245,30 @@ public static void DeleteExcessSaves(string nameStartsWith, int maximumCount)
         }
     }
 
+    public static List<string> CleanUpOldSaves(string nameStartsWith)
```

Comment:
This method should be documented.

Reply:
And if you keep this having the nameStartsWith parameter, this should be renamed to `CleanUpOldSavesOfType`